faq.md

```diff
@@ -5,64 +5,150 @@ layout: faq
 
-- Parse the website to find interesting tutorials
-- Find a Galaxy instance to use for the tutorial 
-- Take one of our interactive tour directly on a Galaxy instance
+## What are the tutorials for?
 
-# How can I use these ressources as an instructor?
+These tutorials can be used for learning and teaching how to use Galaxy for general data analysis, and for learning/teaching specific domains such as assembly and differential gene expression analysis with RNA-Seq data.
+
+## What audiences are the tutorials for?
+
+There are two distinct audiences for these materials.
+
+1. **Self-paced individual learners.** These tutorials provide everything you need to learn a topic, from explanations of concepts to detailed hands-on exercises.
+2. **Instructors.** They are also designed to be used by instructors in teaching/training settings.  Slides, and detailed tutorials are provided.  Some tutorials also include computational support such as Docker images that can be used to scale the lessons up to many participants.
+
+## How is the content licensed?
+
+The content of this website is Creative Commons Attribution 4.0 ([License]({{ site.github_repository }}/LICENSE.md)).
+
+## How can I cite this effort?
+
+We wrote an [article](https://www.biorxiv.org/content/early/2018/04/05/225680) about our efforts.
+
+# For Individual Learners
+
+Learning Galaxy and data analysis on your own and at your own pace?  This material is for you.
+
+## Where do I start?
+
+If you are new to Galaxy then start with one of the introductory topics.  These introduce you to concepts that are useful in Galaxy, no matter what domain you are doing analysis in.
+
+If you are already familiar with Galaxy basics and want to learn how to use it in a particular domain (for example, ChIP-Seq), then start with one of those topics.
+
+If you are already well informed about bioinformatics data analysis and you just want to get a feel for how it works in Galaxy than many tutorials include *Instructions for the impatient* sections.
+
+## How do I use this material?
+
+Many topics include slide decks and if the topic you are interested in has slides then start there.  These will introduce the topic and important concepts.
+
+Most of your learning will happen in the next step - the hands-on tutorials.  This is where you'll become familiar with using the Galaxy interface and experiment with different ways to use Galaxy and the tools in Galaxy.
+
+## Where can I run the hands-on tutorials?
+
+To run the hands-on tutorials you need a Galaxy server to run them on.
+
+Some topics have a [Docker](https://www.docker.com/) image that can be installed and run on participants' laptops.  These images contain Galaxy instances that include all tools and datasets used in a tutorial, as well as saved analyses and repeatable workflows that are relevant. You will need to [install Docker](https://docs.docker.com/install/).
```

Comment:
I would put first the public servers (Docker after) and also mentioning that the for each tutorial, a list of possible servers on which the tutorial can run is available (tools available there)